tests/core/cgo/asm/cgoasm_test.go

```diff
@@ -0,0 +1,21 @@
+package asm
```

Comment:
Could you add build constraints here so that this test doesn't break if we start testing it on non-amd64, non-am64 platforms or e.g. Windows?